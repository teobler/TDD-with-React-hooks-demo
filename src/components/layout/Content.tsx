import * as React from "react";
import {combineClass} from "../../utils/combineClass";

export const Content: React.FC<React.HTMLAttributes<HTMLElement>> = (props) => {
    const { className = '', children, ...restProps } = props;

    return (
        <main className={combineClass('layout-content', className)} {...restProps}>
            {children}
        </main>
    );
};
