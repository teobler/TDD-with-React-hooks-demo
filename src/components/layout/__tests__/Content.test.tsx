import {render} from "@testing-library/react";
import {Content} from "../Content";
import * as React from "react";


describe('Content', () => {
    it('should render correctly', () => {
        const {container} = render(<Content/>);
        expect(container).toMatchSnapshot();
    });
});
