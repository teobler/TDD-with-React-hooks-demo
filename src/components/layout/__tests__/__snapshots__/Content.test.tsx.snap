// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content should render correctly 1`] = `
<div>
  <main
    class="prefix-layout-content"
  />
</div>
`;
